lib/stat_tracker.rb

```diff
@@ -1,5 +1,36 @@
+require_relative './game_collection'
+require 'CSV'
+
 class StatTracker
 
-  
+  attr_reader :game_path, :team_path, :game_team_path
+
+  def initialize(game_path, team_path, game_team_path)
+    @game_path = game_path
+    @team_path = team_path
+    @game_team_path = game_team_path
+  end
```

Comment:
Initialize and self.from_csv in the Stat Tracker class are methods common to all the branches we're making, so let's just be aware that this will cause merge conflicts